### DISCUSSION TITLE
STYJ02YM Script

### DISCUSSION THREAD

**Discussion by bartek810** [PiotrMachowski/lovelace-xiaomi-vacuum-map-card#459] [Q&A] [answered]
Hello everyone. I tried to automate my Xiaomi STYJ02YM Vacuum but i have this problem: 
As soon as I create this script and click on Run without Saving it is working perfectly fine: 
![image](https://user-images.githubusercontent.com/70979941/185682395-0ec2c367-1524-4363-8557-39c58bece28c.png)
But when i press Save and reenter the script edition it converts to something like this:
![image](https://user-images.githubusercontent.com/70979941/185682557-336cc955-3e74-48b9-a7cc-80cdd146a956.png)
Here the script will trigger my vacuum, but the coordinates are now wrong so the vacuum will stop and is not going to clean at all.
I tried to remove the "minus" but than the script is not triggering at all with the error:
![image](https://user-images.githubusercontent.com/70979941/185683134-cba7aea0-3e67-4ea7-b272-8926d26168c5.png)
Can anybody tell me how to setup this script? 
Thank you very much in advance

**Comment by PiotrMachowski** [comment, marked as answer]
> But when i press Save and reenter the script edition it converts to something like this:

It is ok, this is just an alternative formatting. Don't remove any "minuses" and it will be ok

**Reply by bartek810** [reply]
But it is not working that way. I get an red error on my vacuum and it is telling me to resume the cleaning. 
//With Pic1 it is working fine with Pic2 it is not working

**Reply by PiotrMachowski** [reply]
It shouldn't be a problem... Have you tried using a version from the second image in dev tools?

**Reply by bartek810** [reply]
Naprawiles telepatycznie. Dziekuje <3

**Reply by PiotrMachowski** [reply]
![image](https://user-images.githubusercontent.com/6118709/185689083-bfbf378f-cd6d-4017-b07f-a24f26bf4786.png)
